themes/digital.gov/layouts/resources/single.html

```diff
@@ -1,73 +1,82 @@
+          <div class="">
+            <div class="grid-row">
+              <div class="grid-col-12" data-edit-this="{{- $path -}}">
+                <p class="breadcrumb">
+                  <a href="{{- "resources/" | absURL -}}" aria-label="Back to Resources landing page">
+                    <svg class="usa-icon dg-icon dg-icon--standard margin-bottom-05" aria-hidden="true" focusable="false">
+                      <use xlink:href="{{ "uswds/img/sprite.svg#arrow_back" | relURL }}"></use>
+                    </svg>
+                    <span>All Resources</span>
+                  </a>
+                </p>
+                {{- if .Params.kicker -}}
+                <p class="kicker">{{- .Params.kicker -}}</p>
+                {{- end -}}
+                {{/* Page Title */}}
+                <h1>{{- .Title | markdownify -}} </h1>
+    
+                {{/* Deck */}}
+                {{- if .Params.deck -}}
+                <div class="deck">{{- .Params.deck | markdownify -}}</div>
+                {{- end -}}
+              </div>
             </div>
-
-            {{- partial "core/page-data.html" (dict "page" . "page_type" "resource" "branch" ($.Scratch.Get "branch")) -}}
-
           </div>
-
-          <div class="grid-col-12 tablet:grid-col-3">
-
-            {{/* Table of contents */}}
-            {{- partial "core/toc.html" . -}}
-
-            {{/* Last updated */}}
-            {{/* {{- partial "last-updated.html" . -}}  */}}
-
-            {{/* Related Communities and Services */}}
-            {{- partial "core/get_related.html" . -}}
-
-            {{/* Topics */}}
-            {{/* {{- partial "core/list-topics.html" . -}} */}}
-
-            {{/* Share Tools */}}
-            {{/* {{- partial "core/get_sharetools.html" . -}} */}}
-
+        </header>
+    
+        <section>
+          <div class="">
+            <div class="grid-row tablet-lg:grid-gap-6">
+    
+              <div class="grid-col-12 tablet:grid-col-12" data-edit-this="{{- $path -}}">
+    
+                {{/* Content */}}
+                <div class="content usa-prose">
+                  {{- .Content -}}
+                </div>
+    
+                {{- partial "core/page-data.html" (dict "page" . "page_type" "resource" "branch" ($.Scratch.Get "branch")) -}}
+    
+              </div>
+    
+              <!-- <div class="grid-col-12 tablet:grid-col-3">
+    
+                {{/* Table of contents */}}
+                {{- partial "core/toc.html" . -}}
+    
+                {{/* Last updated */}}
+                {{/* {{- partial "last-updated.html" . -}}  */}}
+    
+                {{/* Related Communities and Services */}}
+                {{- partial "core/get_related.html" . -}}
+    
+                {{/* Topics */}}
+                {{/* {{- partial "core/list-topics.html" . -}} */}}
+    
+                {{/* Share Tools */}}
+                {{/* {{- partial "core/get_sharetools.html" . -}} */}}
+    
+              </div> -->
```

Comment:
@nick-mon1 we should try keeping these previous components. It will take some customization, but it _seems_ possible.

See https://github.com/GSA/digitalgov.gov/pull/5800/commits/9cffdb10cd08d841bee42afc5d04bc6b5eed704e

Reply:
We should also fix this extra margin on the promo wrapper

![image](https://user-images.githubusercontent.com/3385219/217663143-6e596f0d-e0ad-43cd-bfe7-505cbbeebba3.png)
https://github.com/GSA/digitalgov.gov/blob/9cffdb10cd08d841bee42afc5d04bc6b5eed704e/themes/digital.gov/src/scss/new/_promos.scss#L19